### ISSUE TITLE
Feature Request : Bookmark all tabs

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#3967]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Requested by https://linkbubble.zendesk.com/agent/tickets/5100 . This feature exists in many modern browsers.

**Expected behavior:**
Bookmark all open tabs.

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version:
> 0.12.1

**Comment by alexwykoff** [comment]
cc @bsclifton

**Comment by phazer-11** [comment]
I vaguely remember seeing this thread now. Forgot about it when I made the feature request on the brave forums after doing a search there. Hoping it gets seen there.

**Comment by hugothebear** [comment]
This would be extremely useful to me also, have spent some time looking for the feature and found this thread. Hope it gets added.

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/saving-open-tabs-to-bookmarks-by-topic-or-date/11609?u=eljuno

**Comment by wesleydean** [comment]
I'd love this feature to be live as well! such a big feature with everyday multitab browsing

**Comment by LaurenWags** [comment]
+1 from community: https://community.brave.com/t/missing-bookmark-all-tabs/17166

**Comment by NumDeP** [comment]
+1

**Comment by EPCarter** [comment]
Agreed, it would be pretty helpful. Maybe as a hotkey or additional option on the Edit Bookmark window.

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/add-all-open-tabs-to-bookmarks-as-new-folder/26221?u=eljuno

**Comment by stelita** [comment]
When is this happening?

**Comment by chaaridev** [comment]
+1

**Comment by cryptogoth** [comment]
+1
